subprojects/packagefiles/cfitsio/meson.build

```diff
@@ -0,0 +1,270 @@
+project(
+  'cfitsio',
+  'c',
+  version: '4.5.0',
+  meson_version: '>=1.1.0',
+  license: 'CFITSIO',
+  license_files: ['licenses/License.txt'],
+)
+has_fortran = add_languages(
+  'fortran',
+  native: false,
+  required: get_option('fortran'),
+)
+
+SOVERSION = 10
+
+cc = meson.get_compiler('c')
+libm = cc.find_library('m')
+
+if target_machine.system() != 'windows'
+  curl = dependency(
+    'libcurl',
+    required: get_option('curl'),
+    fallback: ['curl', 'libcurl'],
```

Comment:
Fallback is probably not necessary since curl wraps can just `[provide] curl = curl_dep`.

Reply:
removed